### PR TITLE
Update Node CLI commands

### PR DESCRIPTION
**Pull request by YazzyYaz**
Updates docs for this issues https://github.com/celestiaorg/celestia-node/commit/51925630b5186f43a212c6d1e96d9acdf68591f8

### CODE REVIEW

**Review by rootulp — DISMISSED** [review]
https://github.com/celestiaorg/celestia-node/commit/51925630b5186f43a212c6d1e96d9acdf68591f8 was merged 11 days ago and doesn't appear in a release yet: https://github.com/celestiaorg/celestia-node/releases

If we update these docs to use new node CLI flags, then we should update https://github.com/celestiaorg/docs/blob/main/docs/developers/celestia-node.md?plain=1#L16= to use `main` because `tags/v0.3.0-rc2` won't have these CLI flags.

Alternatively (based on celestia-node team's preference) we wait until a new release is cut with the CLI flags and upgrade to that version in this PR.

**Review by renaynay — CHANGES_REQUESTED** [review]
Looks good but three things to note: 

1. we should mention that if `--core.grpc.port` is not provided, it defaults to `9090`
2. if `--core.rpc.port` is not provided, it defaults to `26657`
3. since leading `http` `https`, etc prefixes get trimmed from ip, we should have it be `--core.ip <ip>` everywhere

So technically `--core.grpc.port` doesn't have to be provided anywhere unless the port deviates from the default which is `9090`

File: docs/nodes/full-storage-node.md [hunk 1]
```diff
@@ -49,9 +49,11 @@ Start the Full Storage Node with a connection to a validator node's gRPC endpoin
   or query for the node's account balance, a gRPC endpoint of a validator
   (core) node must be passed as directed below.
 
+<!-- markdownlint-disable MD013 -->
 ```sh
-celestia full start --core.grpc http://<ip addr of core node>:9090
+celestia full start --core.ip http://<ip-address> --core.grpc.port 9090 --core.rpc.port 26657
```
Comment (renaynay) [inline docs/nodes/full-storage-node.md:54]: Does not need `--core.rpc.port` for full node

**Review by renaynay — CHANGES_REQUESTED** [review]

File: docs/nodes/systemd.md [hunk 2]
```diff
@@ -167,7 +167,7 @@ After=network-online.target
 
 [Service]
 User=$USER
-ExecStart=$HOME/go/bin/celestia light start --core.grpc http://<ip>:9090
+ExecStart=$HOME/go/bin/celestia light start --core.ip <ip> --core.grpc 9090
```
Comment (Wondertan) [inline docs/nodes/systemd.md:170]: Small miss it seems
```suggestion
ExecStart=$HOME/go/bin/celestia light start --core.ip <ip> --core.grpc.port 9090
```

Comment (Wondertan) [reply docs/nodes/systemd.md:170]: Or we can avoid passing 9090 at all as its a default value.

**Review by Wondertan — COMMENTED** [review]
Note that passing `9090` to `--core.grpc.port` is not required as it's a default value and passed automatically. Similarly, for `--core.rpc.port`, the `26657` is a default value; hence the flag can be omitted. Should we pass those or not? I don't have strong preferences and think you know better what would be the best.

File: docs/nodes/light-node.md [hunk 3]
```diff
@@ -146,16 +146,22 @@ is usually exposed on port 9090):
...
 
+For ports:
+
+> NOTE: The `--core.grpc.port` defaults to 9090, so if you do not specify
+  it in the command line, it will default to that port. You can use the flag
+  to specify another port if you prefer.
+
 ```sh
-celestia light start --core.grpc http://<ip>:9090
+celestia light start --core.ip <ip> --core.grpc.port 9090
```
Comment (renaynay) [inline docs/nodes/light-node.md:156]:
```suggestion
celestia light start --core.ip <ip-address> --core.grpc.port <port>
```

File: docs/nodes/full-storage-node.md [hunk 4]
```diff
@@ -81,9 +89,11 @@ In order to run a full storage node using a custom key:
    at the correct path (default: `~/.celestia-full/keys/keyring-test`)
 2. The name of the custom key must be passed upon `start`, like so:
 
+<!-- markdownlint-disable MD013 -->
 ```sh
-celestia full start --core.grpc http://<ip>:9090 --keyring.accname <name_of_custom_key>
+celestia full start --core.ip http://<ip-address> --core.grpc.port 9090 --keyring.accname <name_of_custom_key>
```
Comment (renaynay) [inline docs/nodes/full-storage-node.md:94]:
```suggestion
celestia full start --core.ip http://<ip-address> --core.grpc.port <port> --keyring.accname <name-of-custom-key>
```

File: docs/nodes/full-storage-node.md [hunk 5]
```diff
@@ -53,9 +53,17 @@ Start the Full Storage Node with a connection to a validator node's gRPC endpoin
...
+A note on ports:
+
+> NOTE: The `--core.grpc.port` defaults to 9090, so if you do not specify
+  it in the command line, it will default to that port. You can use the flag
+  to specify another port if you prefer.
+
+<!-- markdownlint-disable MD013 -->
 ```sh
-celestia full start --core.grpc http://<ip addr of core node>:9090
+celestia full start --core.ip http://<ip-address> --core.grpc.port 9090
```
Comment (renaynay) [inline docs/nodes/full-storage-node.md:64]:
```suggestion
celestia full start --core.ip http://<ip-address> --core.grpc.port <port>
```

File: docs/nodes/bridge-node.md [hunk 6]
```diff
@@ -76,25 +76,28 @@ Follow the tutorial for installing Celestia Node [here](../developers/celestia-n
 Run the following:
 
 ```sh
-celestia bridge init --core.remote tcp://<ip-address>:26657
+celestia bridge init --core.ip <ip-address> --core.rpc.port 26657
```
Comment (renaynay) [inline docs/nodes/bridge-node.md:79]:
```suggestion
celestia bridge init --core.ip <ip-address> --core.rpc.port <port>
```

File: docs/nodes/bridge-node.md [hunk 7]
```diff
@@ -76,25 +76,28 @@ Follow the tutorial for installing Celestia Node [here](../developers/celestia-n
...
 (which is usually exposed on port 9090):
 
-> NOTE: In order for access to the ability to get/submit state-related information,
-  such as the ability to submit PayForData transactions, or query for the node's
-  account balance, a gRPC endpoint of a validator (core) node must be passed as
-  directed below._
-
 ```sh
-celestia bridge start --core.grpc http://<ip>:9090
+celestia bridge start --core.ip <ip> --core.grpc.port 9090
```
Comment (renaynay) [inline docs/nodes/bridge-node.md:94]:
```suggestion
celestia bridge start --core.ip <ip-address> --core.grpc.port <port>
```

File: docs/developers/node-tutorial.md [hunk 8]
```diff
@@ -119,20 +119,25 @@ an example public Core Endpoint.
   purposes. You can find a list of RPC endpoints [here](../nodes/mamaki-testnet#rpc-endpoints)
 
 ```sh
-celestia light start --core.grpc http://<ip-address>:9090
+celestia light start --core.ip <ip> --core.grpc.port 9090
```
Comment (renaynay) [inline docs/developers/node-tutorial.md:122]:
```suggestion
celestia light start --core.ip <ip-address> --core.grpc.port <port>
```

**Review by renaynay — CHANGES_REQUESTED** [review]

**Review by renaynay — APPROVED** [review]
lgtm

**Comment by renaynay** [comment]
Looks like lint failed though

File: docs/developers/node-tutorial.md [hunk 9]
```diff
@@ -119,20 +119,25 @@ an example public Core Endpoint.
...
-celestia light start --core.grpc https://rpc-mamaki.pops.one:9090
+celestia light start --core.ip https://rpc-mamaki.pops.one --core.grpc.port 9090
 ```
 
 ### Keys and wallets
 
 You can create your key for your node by running the following command:
 
 ```sh
+celestia light start --core.ip <ip-address> --core.grpc.port <port> --keyring.accname developer
```
Comment (renaynay) [inline docs/developers/node-tutorial.md:140]: ?

Comment (renaynay) [reply docs/developers/node-tutorial.md:140]: I think something happened to the old key instruction docs?

**Review by renaynay — CHANGES_REQUESTED** [review]

File: docs/developers/node-tutorial.md (hunk 9 above)
Comment (YazzyYaz) [reply docs/developers/node-tutorial.md:140]: @renaynay can you provide me context on what happened?

Comment (YazzyYaz) [reply docs/developers/node-tutorial.md:140]: Like, what changes are you specifically requesting here?

Comment (renaynay) [reply docs/developers/node-tutorial.md:140]: The command is to start a light node, not to create a key

**Review by renaynay — APPROVED** [review]

**Review by adlerjohn — APPROVED** [review]
utACK

**Pull request merged**